pynws/nws.py

```diff
@@ -59,9 +59,9 @@ def __init__(
 
     async def get_points_stations(self: Nws) -> List[str]:
```

Comment:
I didn't rename this function to avoid a breaking API change.

Reply:
I think this is fine.  This function still still use the points if the gridpoints are unavailable.  Documentation is one of the weak points of this package, but you could consider briefly adding this info to the docstring.  Not a requirement.